### ISSUE TITLE
[P3][Sourcing][Accessibility] Platform Table search refresh icon doesn't have tooltip

### ISSUE THREAD

**Issue by I543348** [SAP/fundamental-ngx#14112]
### Describe your proposal

The ATL team reported that refresh button in search does not have a tooltip for disabled users and is an accessibility issue.
replicable : https://docs-0-57--fundamental-ngx.netlify.app/#/platform/table/settings-dialog-table#combined-settings

### Which versions of Angular and Fundamental Library for Angular are affected?

angular 20 , NGX platform: 0.57.6

### Steps to Reproduce (for bugs)

Hover over refresh button in search field using mouse and see if tooltip comes
https://docs-0-57--fundamental-ngx.netlify.app/#/platform/table/settings-dialog-table#combined-settings 

### StackBlitz example(s)

https://docs-0-57--fundamental-ngx.netlify.app/#/platform/table/settings-dialog-table#combined-settings

### Priority

Medium

### Declaration

- [x] I'm not disclosing any internal or sensitive information.
